cylc/uiserver/data_store_mgr.py

```diff
@@ -232,17 +294,17 @@ async def entire_workflow_update(self, ids=None):
             ids = []
 
         # Prune old data
-        for w_id in list(self.data):
-            if w_id not in self.workflows_mgr.workflows:
-                del self.data[w_id]
+        # for w_id in list(self.data):
+        #     if w_id not in self.workflows_mgr.active:
+        #         del self.data[w_id]
```

Comment:
Can probably get rid of this .. Was just a cheap way of trimming the data-store of workflows, but even new syncs replace the target workflows with new data (see code below)..